src/Components/Subscriptions/Modal/modal.module.css

```diff
@@ -0,0 +1,24 @@
+.modalContainer {
+  position: absolute;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 200vh;
+  width: 300vw;
+  background-color: rgba(0, 0, 0, 0.5);
+}
+
+.modalContent {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: 5%;
+  height: 8%;
```

Comment:
this parameter should disturb the message box.
maybe you can fix it by removing it and modify margin or padding if necessary.